oolushola/Banka
# #[165169661] Unit test create bank account api endpoint

### What does this PR do?
Test assertions for user to create bank account

### Description of Task to be completed?
Have the following resource endpoints tested with tdd to check if all assertions are met.

- POST:/api/v1/user/create-bank-account/<user id>

### How should this be manually tested?
After cloning the repo, cd into it and from the terminal RUN 'npm run test' to check the build status of the endpoint

### Any background context you want to provide?
NA

### What are the relevant pivotal tracker stories?
[#165169661](https://www.pivotaltracker.com/story/show/165169661)

### Screenshots (if appropriate)
NA

### Questions:
NA
